test/DomainController.ts

```diff
@@ -0,0 +1,104 @@
+import { ethers } from "hardhat";
+import { Contract, ContractFactory } from "ethers";
+import { expect } from "chai";
+
+describe("DomainController", function () {
+  let DomainController: ContractFactory;
+  let domainController: Contract;
+  let owner: string;
```

Comment:
// line 8-10 
// wrong data type for owner => should be SignerWithAddress
// - 1 point